nanoFramework.Json.Test/JsonUnitTests.cs

```diff
@@ -173,10 +73,22 @@ public void Can_serialize_and_deserialize_arrays_of_class_objects()
                 }
             };
 
+            var startTimestamp = Environment.TickCount64;
+
             var result = JsonConvert.SerializeObject(myTown);
-            Debug.WriteLine($"Serialized Array: {result}");
+
+            var endTimestamp = Environment.TickCount64;
+
+            Debug.WriteLine($"Serialized Array: {result} took {endTimestamp - startTimestamp}ms");
```

Comment:
in this case, I would use OutputHelper so even in Release we will have this info

Reply:
Agreed! @Feiko I've fixed this one. Make sure to pull before doing any changes.